### DISCUSSION TITLE
Copilot not responding to chats in IDE

### DISCUSSION THREAD

**Discussion by fizarazvi** [community/community#132804] [Copilot Conversations]
I recently got my copilot (for education), but in the 
IDE it keeps buffering and never gives an answer to the chat

![image](https://github.com/user-attachments/assets/d878daf7-3dbe-4f51-8688-4f9db0099bf6)

**Comment by Akash1134** [comment]
Hi @fizarazvi ,

Could you please share more details along with the version of your IDE & Copilot? What we have seen is that these issues are caused by either a network/proxy issue or by security software on your client blocking traffic. Do you connect to the internet directly or via a proxy? I'd like to check your connection to the Copilot Proxy?

Whenever you can -- gather the logs from your IDE and share in the conversation: [Steps to gather logs](https://docs.github.com/en/copilot/troubleshooting-github-copilot/viewing-logs-for-github-copilot-in-your-environment?tool=jetbrains#collecting-log-files)

Thanks!

**Comment by fizarazvi** [comment]
Hi @Akash1134 ,

I tried re-login and it fixed my error after sometime on its own, not sure how it happened. 
I'm usually connected to VPN

Thank you!
